.github/workflows/main.yaml

```diff
@@ -81,7 +81,7 @@ jobs:
   # Builds, tags, and pushes Cobalt docker build images to ghr.
   docker-build-image:
     needs: [initialize]
-    runs-on: [self-hosted, chrobalt-linux-runner]
+    runs-on: [self-hosted, cobalt-linux-runner]
```

Comment:
This may break - unless Igor re-deployed new runners with new names.

It's a whole other infra / capacity pool, so this name isn't trivial to change. Same for all runner names. I wouldn't worry about renaming those now.

CC @isarkis

Reply:
done